### DISCUSSION TITLE
Can I host pocketbase on hostinger?

### DISCUSSION THREAD

**Discussion by ammarrik** [pocketbase/pocketbase#1664] [Q&A]
Is there any specific guide to host pocketbase on hostinger?

**Comment by ganigeorgiev** [comment]
If you are using a shared hosting plan, then most likely it will not be possible to host PocketBase.

But if your plan allows running an external application and opening ports (aka. you have a VPS), then the deployment should be similar to the one described in the "Minimal setup" guide - https://pocketbase.io/docs/going-to-production/#minimal-setup.
